Python/import.c

```diff
@@ -1760,6 +1782,23 @@ import_find_and_load(PyThreadState *tstate, PyObject *abs_name)
     return mod;
 }
 
+PyObject *
+PyImport_GetModule(PyObject *name)
+{
+    PyThreadState *tstate = _PyThreadState_GET();
+    PyObject *mod;
+
+    mod = import_get_module(tstate, name);
+    if (mod != NULL && mod != Py_None) {
+        if (import_ensure_initialized(tstate, mod, name) < 0) {
+            Py_DECREF(mod);
+            remove_importlib_frames(tstate);
```

Comment:
What made you add this `remove_importlib_frames()` call?  (I'm sure it's correct, BTW.)

Reply:
@serhiy-storchaka had requested it -- I think most likely because it's found in the corresponding `error:` path in `PyImport_ImportModuleLevelObject`.